src/iframe-helper.js

```diff
@@ -380,3 +380,61 @@ function parseIfNeeded(data) {
   }
   return data;
 }
+
+
+
+/**
+ * Manages a postMessage API for an iframe with a subscription message and
+ * a way to broadcast messages to all subscribed windows, which
+ * in turn must all be descendants of the contentWindow of the iframe.
+ */
+export class SubscriptionApi {
```

Comment:
Well, we do need to keep the state so that the send() method can be used. We could consider hiding the class away and returning send() as a function closure, but that just obscure that fact that there is state and if we ever want to add destroy() to clean up state we'd need to revert back to exposing the class. I could do that if you feel strongly, but IMHO a class is the right answer here.